lib/teleterm/apiserver/handler/handler_access_requests.go

```diff
@@ -149,5 +164,17 @@ func newAPIAccessRequest(req clusters.AccessRequest) *api.AccessRequest {
 		SuggestedReviewers: req.GetSuggestedReviewers(),
 		ThresholdNames:     thresholdNames,
 		ResourceIds:        requestedResourceIDs,
+		Resources:          resources,
+	}
+}
+
+// resourceIDToString marshals a ResourceID to a string.
+func resourceIDToString(id *api.ResourceID) string {
```

Comment:
Why not add a `func String() string` method to `ResourceID` instead?

Reply:
I see that, but if you follow the signature suggested above, `ResourceID` will implement the `fmt.Stringer` interface and it will display correctly in log messages, etc.

Reply:
I probably should have done that for `types.ResourceID`, but it doesn't look like we currently add any custom methods to teleterm generated types, I think we'd have to put them in the `protogen/golang/v1` package?